gremlin-dotnet/src/Gremlin.Net/Gremlin.Net.csproj

```diff
@@ -48,7 +48,7 @@ Please see the reference documentation at Apache TinkerPop for more information
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
I don't understand the need to bump the json dependency of the actual Gremlin.Net library.

Reply:
The reason was that this version of Newtonsoft.Json couldn't deserialize _TooBigInteger_ according to @danielcweber and I didn't see a good reason why we shouldn't update the dependency.
Are you against the version bump?

Reply:
That's why I wanted to upgrade Newtonsoft.Json for the next release anyway.